### ISSUE TITLE
Better support for first-time deployments

### ISSUE THREAD

**Issue by rogeruiz** [18F/cloud-foundry-cli#8]
Because `cf_deploy.sh` only uses the Autopilot plugin, apps will fail to push if it's the first time they're being deployed. The deployment script should support the scenario when the application doesn't already exist in the supplied org and space.

**Comment by adborden** [comment]
I've done this before with something like
```
command=push
if cf app $app; then
  command=zero-downtime-push
fi

cf $command ...
```

**Comment by ccostino** [comment]
We have too, @adborden.  PR incoming! :-)

**Comment by rogeruiz** [comment]
Closed by #14
